sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -89,10 +89,10 @@
       <Link>Properties\AssemblyInfo.Shared.cs</Link>
     </Compile>
     <Compile Include="Helpers\*.cs" />

```

Comment:
I am fine to keep it like this but optionally we could use: `Include="**\*.cs" Exclude="**\obj\**\*;**\*.Designer.cs"`. We can omit the `Designer.cs` part if there is no resx in this project..